README.md

```diff
@@ -54,13 +54,34 @@ Each row represents a mass spectrometry run in one of the formats: raw, RAW, mzM
 Now, you can run the pipeline using:
 
 ```bash
-nextflow run nf-core/mhcquant
+nextflow run nf-core/mhcquant \
     -profile <docker/singularity/.../institute> \
     --input 'samplesheet.tsv' \
     --fasta 'SWISSPROT_2020.fasta' \
     --outdir ./results
 ```
 
+Optional parameters for additional functionality:
```

Comment:
Use only one additional example of a pipeline run to show quanitfy, global_fdr, and generate_speclib